app/models/folder.rb

```diff
@@ -2,6 +2,10 @@ class Folder < ApplicationRecord
   validates :name, presence: true
   validates :route, presence: true
 
+  validates :route, uniqueness: true
```

Comment:
fix this. Have to validate by url, not route otherwise two people cant have the same route